src/test/java/org/broadinstitute/consent/http/service/ResearcherServiceTest.java

```diff
@@ -1,11 +1,7 @@
 package org.broadinstitute.consent.http.service;
 
 import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.atLeast;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyZeroInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Avoid * imports. In Intellij, set the class count and names count boxes to 9999 and that should fix what it does when optimizing imports: 

![Screen Shot 2020-09-08 at 12 50 45 PM](https://user-images.githubusercontent.com/116679/92505352-f4752c00-f1d1-11ea-8006-1f3c2feb4eab.png)